HighTeach/urls.py

```diff
@@ -19,5 +19,6 @@
 
 urlpatterns = [
     path('admin/', admin.site.urls),
-    path('', views.homePage, name='homePage')
+    path('', views.homePage, name='homePage'),
+    path('logout/', views.Logout, name='Logout')
```

Comment:
This is temporary, right? If not we should use `LogoutView`

Reply:
The option to logout is not temporary (only the way to log in through the admin), I changed it